.github/workflows/main.yml

```diff
@@ -0,0 +1,91 @@
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  #
+  # Ensure the debian package can be built and installed
+  # without any errors. Note that we require python3 to
+  # be installed.
+  #
+  deb-build-and-test-run:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.6'
+      #
+      # Needed for all the debian file related utilities.
+      #
+      - run: sudo apt install debhelper devscripts equivs
```

Comment:
nit: `apt-get` is recommended over `apt` in scripts.
Also, might want to add `-y`